### PR TITLE
Schedule filter

### PR DESCRIPTION
**Pull request by schester77**
Would love some feedback on this. I've restructured the schedule to allow for 'day' and 'week' views. The parent component that is used in index.js is <Schedule>. Within <Schedule>, there is some state that controls whether <WeeklySchedule> or <DaySchedule> is displayed. Each of these use <WeeklyShow> or <DayShow> components, respectively. You click on the headings ("monday, tuesday, etc.") to filter by day. 

Also, ShowDetails is not yet implemented, but that can happen pretty easily once this is merged.

### CODE REVIEW

File: src/components/DaySchedule.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+import DayShow from "./DayShow";
+import { getDayString } from "../lib/component-utils.js";
+import styles from "../styles/DaySchedule.module.css";
+import { showType } from "../lib/types.js";
+
+export default function DaySchedule({ shows , day}){
+  const showList = shows.filter((s) => getDayString(s.time.day) === day)
+  const showListSorted = showList.sort((a, b)=> {return a.time.hour - b.time.hour});
+
+  //this is stupid code: but its the only solution I can come up with...
```
Comment (ethansaxenian) [inline src/components/DaySchedule.js]: I think the fix is really simple! Try adding `flex-wrap: wrap;` to the CSS inline class. Then you shouldn't need multiple different lists.

File: src/styles/DaySchedule.module.css [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+  padding: 5px;
+  text-align: left;
+}
+.showListItem {
+  cursor: pointer;
+  width: fit-content;
+}
+.inline {
+  display: flex;
+  flex-direction: row;
```
Comment (ethansaxenian) [inline src/styles/DaySchedule.module.css:15]: See comment in `DaySchedule.js`: try adding `flex-wrap: wrap;` here!

**Review by ethansaxenian — APPROVED** [review]
Awesome job Sam! I love the use of `ScheduleFilter`.

I think there's a line of CSS that will make the formatting of `DaySchedule` much simpler! See my comments

**Review by maddievb — APPROVED** [review]
looks great! ditto on ethan's comment and also with routing we'll be able to use links to get to the show details page which is actually a lot easier!! (2/3)

**Review by ethanberner — APPROVED** [review]
awesome job! one small css comment: when a user navigates to the schedule from home, the cursor doesn't become a pointer until after the first click on a day of the week, so the filter functionality might be a bit hard to see. 3/3

**Pull request merged**